.github/workflows/Cosmo.Mac.Ubuntu.Window.go.yml

```
@@ -0,0 +1,42 @@
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v5.2.0
+      with:
+        go-version: '1.20'  # You can specify a version range or use 'stable' if you want the latest stable version
+        cache: true
+        cache-dependency-path: go.sum
+
+    - name: Fetch Latest Cosmos SDK Version
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:42: Double quote to prevent globbing and word splitting

(shellcheck)